semantic/src/site/elements/label.overrides

```diff
@@ -30,6 +30,11 @@
   padding-left: 0;
 }
 
+.ui.label.link-label {
+  .ui.label.simple-label();
+  color: @interactiveHigh;
+}
```

Comment:
Is `.link-label` one of our classes, or part of Semantic? If the latter, then are you sure this won't effect any other labels we're using?

Reply:
It's our classes, used to display the Show History as a link to open the HistoryPanel modal